src/Trakx.YouSign.ApiClient/openApi3.yaml

```diff
@@ -0,0 +1,1167 @@
+swagger: '2.0'
```

Comment:
👍 I hope this was not too painful to generate or that they gave it all ready!
how did you do? thanks a lot! 🎉

Reply:
It was a little. The steps I had to execute were:
1) To generate the openApi spec based on swagger spec (via https://app.swaggerhub.com/);
2) To keep only the methods and entities required for the integration + 2 additional methods just for testing (list organizations and list users);
3) To remove prop "metadata" from some entities, it was breaking the live load.